frontend/app/src/index.tsx

```diff
@@ -28,6 +28,9 @@ import ThemedApp from "./ThemedApp"
 
 const engine = new Styletron({ prefix: "st-" })
 
+// Timestamp when React is loaded for GUEST_READY message
+window.streamlitReactLoad = Date.now()
```

Comment:
[nit] should this go even earlier - above line 29. Or even above the imports?

Reply:
`streamlitExecutionStartedAt`?

Reply:
This could be a prop passed to Themed app. But seems ok to use a window variable.

Reply:
If its possible to pass somewhere and not having it as a window variable would be even nicer (for testing, reasoning of code flow etc.) 🙂 👍